learn/filtering_and_sorting/geosearch.mdx

```diff
@@ -37,6 +37,37 @@ When using JSON and NDJSON, `_geo` must contain an object with two keys: `lat` a
 }
 ```
 
+`_geojson` must be an object whose contents follow the [GeoJSON specification](https://geojson.org/):
+
+```json
+{
+  …
+  "_geojson": {
+    "type": "Feature",
+    "geometry": {
+      "type": "Point",
+      "coordinates": [0.0, 0.0]
+    }
+  }
+}
+```
+
+Meilisearch does not support transmeridian shapes. If your document includes a transmeridian shape, split it into two separate shapes grouped as a `MultiPolygon` or `MultiLine`. Transmeridian shapes are polygons that cross the 180th or antimeridian.
+
+Meilisearch does not support polygons with holes. If your polygon consists of an external ring and an inner empty space, Meilisearch ignores the hole and treats the polygon as a solid shape.
```

Comment:
Maybe we should put that in a warning or anything that stands out. What do you think? 🤔 
I'm afraid it could be a complex to understand source of bug 🤔